src/main/java/duke/Deadlines.java

```diff
@@ -0,0 +1,20 @@
+package duke;
+
+public class Deadlines  extends Task {
```

Comment:
Suggest to change class name to Deadline, since each instance of this class represents a single Deadline.